src/NzbDrone.Core/Tags/TagService.cs

```diff
@@ -109,7 +115,8 @@ public List<TagDetails> Details()
                         ImportListIds = importLists.Where(c => c.Tags.Contains(tag.Id)).Select(c => c.Id).ToList(),
                         NotificationIds = notifications.Where(c => c.Tags.Contains(tag.Id)).Select(c => c.Id).ToList(),
                         RestrictionIds = restrictions.Where(c => c.Tags.Contains(tag.Id)).Select(c => c.Id).ToList(),
-                        ArtistIds = artists.Where(c => c.Tags.Contains(tag.Id)).Select(c => c.Id).ToList()
+                        ArtistIds = artists.Where(c => c.Tags.Contains(tag.Id)).Select(c => c.Id).ToList(),
+                        RootFolderIds = rootFolders.Where(c => c.DefaultTags.Contains(tag.Id)).Select(c => c.Id).ToList()
```

Comment:
Need to also update `QualityProfileService.cs` and `MetadataProfileService.cs`. Both of their `Delete` functions check for in use prior to allowing deletion. Delete should be prevented if a root folder is using them.